src/azure-cli/azure/cli/command_modules/storage/azcopy/util.py

```diff
@@ -53,8 +53,7 @@ def install_azcopy(self, install_location):
         else:
             raise CLIError('Azcopy ({}) does not exist.'.format(self.system))
         try:
-            os.chmod(install_dir,
-                     os.stat(install_dir).st_mode | stat.S_IWUSR | stat.S_IWGRP | stat.S_IWOTH)
+            os.chmod(install_dir, os.stat(install_dir).st_mode | stat.S_IWUSR)
```

Comment:
It seems `stat.S_IWUSR` is set by default:

```py
import os
import stat
install_dir = 'temp'
os.makedirs(install_dir)
print(bool(os.stat(install_dir).st_mode & stat.S_IWUSR))
```

Output:

```
True
```

`stat.S_IWUSR` is not explicitly set when downloading `bicep` CLI:

https://github.com/Azure/azure-cli/blob/9464462b8b20717a908ca7a70b7c9ad6f202897a/src/azure-cli/azure/cli/command_modules/resource/_bicep.py#L104-L106